src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -168,11 +168,9 @@ private void insertDrivers(Car car) {
     }
 
     private Car getCar(ResultSet resultSet) throws SQLException {
-        Car car = new Car();
         Manufacturer manufacturer = getManufacturer(resultSet);
+        Car car = new Car(resultSet.getString("model"), manufacturer, Collections.emptyList());
```

Comment:
pleas add also additional a constructor without drivers as parameter, so you don't need pass an empty list here.